File: 8-dolar-real/conversao.py
valor_dolar = float(input("Digite o valor em dólares (US$): "))
cotacao_dolar = float(input("Digite a cotação do dólar: "))

valor_real = valor_dolar * cotacao_dolar
print(f"O valor em reais (R$) é: {valor_real:.2f}")

File: 8-dolar-real/conversao.go
package main

import "fmt"

func main() {
	var valorDolar, cotacaoDolar, valorReal float64
	fmt.Println("Digite o valor em dólares (US$): ")
	fmt.Scan(&valorDolar)
	fmt.Println("Digite a cotação do dólar: ")
	fmt.Scan(&cotacaoDolar)

	valorReal = valorDolar * cotacaoDolar
	fmt.Printf("O valor em reais (R$) é: %.2f\n", valorReal)
}
